src/MOIWrapper.jl

```diff
@@ -374,3 +374,33 @@ end
 function LQOI.delete_variables!(instance::Optimizer, start_col::Int, end_col::Int)
     delete_columns(instance.inner, [Cint(i-1) for i in start_col:end_col])
 end
+
+# Corresponds to the `Status` struct defined in https://github.com/coin-or/Clp/blob/master/Clp/src/ClpSimplex.hpp#L114.
```

Comment:
Use a link to a specific commit instead of to the master branch. Line numbers can change in the future.